### ISSUE TITLE
Shrine - Improve Focus support

### ISSUE THREAD

**Issue by perclasson** [material-components/material-components-flutter-gallery#567]
## Acceptance Criteria
* You should be able to select a product with Focus
* You should be able to select a category with Focus

**Comment by perclasson** [comment]
This is duplicate of https://github.com/material-components/material-components-flutter-gallery/issues/316
